assets/js/amp-block-validation.js

```diff
@@ -139,33 +138,6 @@ const ampBlockValidation = ( function() { // eslint-disable-line no-unused-vars
 			return false;
 		},
 
-		/**
-		 * Handles admin notices for AMP Stories.
-		 */
-		handleAMPStoryChange: function handleAMPStoryChange() {
```

Comment:
This notice at the top of the entire editor is replaced by a notice above the Featured Image control in `with-featured-image-notice.js`.